deploy/deploy.sh

```diff
@@ -86,7 +91,12 @@ done
 echo "postgress is ready"
 
 # Create spiceDB bootstrap schema configmap
-oc create configmap spicedb-schema --from-file=schema.yaml -n $NAMESPACE
+
+if ! oc get configmap spicedb-schema >/dev/null 2>&1; then
+    oc create configmap spicedb-schema --from-file=schema.yaml -n $NAMESPACE
+else
+    echo "Configmap 'spicedb-schema' already exists."
+fi
```

Comment:
Probably the idiomatic way to do this would be to apply a template with (oc apply ...), which is idempotent, rather than use oc create, which complains if it exists, but if this is tested, then probably just go with it. :)

Reply:
It looks that I would need to create resource definition file like - schema yml is just spicedb schema.
```
apiVersion: v1
kind: ConfigMap
metadata:
...
```

I prefer to do it in follow PR